api/server.go

```diff
@@ -204,6 +205,58 @@ func (s *server) walletSignHandler(jc jape.Context) {
 	}
 }
 
+func (s *server) walletSplitHandler(jc jape.Context) {
+	// parse request
+	var wfr WalletSplitRequest
+	if jc.Decode(&wfr) != nil {
+		return
+	}
+	if wfr.Amount.Cmp(types.SiacoinPrecision) < 0 {
+		jc.Error(errors.New("'amount' has to be at least 1SC"), http.StatusBadRequest)
+		return
+	}
+	if wfr.Outputs == 0 {
+		jc.Error(errors.New("'outputs' has to be greater than zero"), http.StatusBadRequest)
+		return
+	}
+
+	// build the transaction
```

Comment:
I considered adding a method to the wallet interface that builds the transaction but while writing it I realised it was calling internal methods anyway. I think it makes sense to keep these 6-8 lines in the handler and then reuse the functionality of `FundTransaction` and `SignTransaction` to do the work for us? 

This is not in line with the implementation of the route in `walletd`, but I wanted to discuss it first because the response object is totally different. I'm personally in favour of funding/signing/broadcasting over returning all necessary transaction parameters, as we do in `walletd`